### DISCUSSION TITLE
Can I list MRU the bookmarks added? (and/or regardless of folder added)

### DISCUSSION THREAD

**Discussion by stdedos** [pfcao/abookmark#17] [Q&A] [answered]
Can I list MRU the bookmarks added? (and/or regardless of folder added)

**Comment by pfcao** [comment, marked as answer]
There are several related features:

1, search - recent (days).
2, search in folder. It is sorted by date.
3, sort nodes by date added.
4, settings - main - app start page - last panel.

You may also set any folder as 'temp folder' or 'add to short cut list', or set as sub-topbar.

**Reply by stdedos** [reply]
> 1, search - recent (days).

Ah, yes, right - that's what I was looking for

![image](https://github.com/user-attachments/assets/47731ad2-6c37-4f06-bd78-fb9877eb2a03)

> 3, sort nodes by date added.

That IIRC is a _permanent_ change (Thank **GOD** you've built Ctrl+Z 😰)
